### ISSUE TITLE
Bot fails to start with Visual Studio in Python 3.7

### ISSUE THREAD

**Issue by CaptainKnuddel** [Nicolas-Klatzer/openhiven.py#68]
**Describe the bug**
Doesnt work

```python
import openhivenpy as hiven

client = hiven.UserClient("TOKEN")

@client.event()
async def on_ready():
    print("Bot is ready")

client.run()
```

Exception:
```python
C:\Users\Lorenz\AppData\Roaming\Python\Python37\site-packages\openhivenpy\gateway\ws.py:235: RuntimeWarning: coroutine 'Websocket.ws_connect.<locals>.ws_connection' was never awaited
  self._connection_task = asyncio.create_task(ws_connection(), name="openhivenpy-ws-connection")
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
C:\Users\Lorenz\AppData\Roaming\Python\Python37\site-packages\openhivenpy\gateway\connection.py:128: RuntimeWarning: coroutine 'Client.__check_if_data_is_complete' was never awaited
  _tasks.append(asyncio.create_task(async_task(), name=task_name))
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
```

**Environment**
Python: *3.7*
OpenHiven.py: *newest*
OS: *Win10*

**Comment by Nicolas-Klatzer** [comment]
okay, that's better and I found the bug now. Asyncio added the name tag feature in 3.8, which causes an error with Python 3.7. Will push a fix for it and release a new bug-fix release

**Comment by Nicolas-Klatzer** [comment]
The RuntimeWarning is here only a warning though and not the real error, but is caused since the program did not correctly execute the task and the program crashed before the program could resolve it

**Comment by Nicolas-Klatzer** [comment]
Closed with #69
